File: clases 25-11-2025/Tipos de clases c#/clases normales/Estudiante.cs
public class Estudiante
{
    public string Nombre { get; set; }
    public int Grado { get; set; }

    // Constructor normal
    public Estudiante(string nombre, int grado)
    {
        Nombre = nombre;
        Grado = grado;
    }

    // Constructor copia
    public Estudiante(Estudiante otro)
    {
        Nombre = otro.Nombre;
        Grado = otro.Grado;
    }
}

File: clases 25-11-2025/tipos de clases Java/clases normales/Estudiante.java
public class Estudiante {
     String nombre;
    int grado;

    // Constructor normal para crear el original (necesario)
    public Estudiante(String nombre, int grado) {
        this.nombre = nombre;
        this.grado = grado;
    }

    // Constructor copia
    public Estudiante(Estudiante otro) {
        this.nombre = otro.nombre;
        this.grado = otro.grado;
    }
}
